### DISCUSSION TITLE
Better printer naming and categorization

### DISCUSSION THREAD

**Discussion by davidzwa** [OctoFarm/OctoFarm#677] [Feature Request]

**Comment by doucettom** [comment]
Avatars, picking up the color from OP's appearance 
menu and location tags are all great ideas IMHO. I'm using color/name combos in OP and avatars in Telegram via Octoprint-Telegram plugin personally and being able to use all the same in OF would be awesome :)

**Reply by NotExpectedYet** [reply]
> Avatars, picking up the color from OP's appearance menu and location tags are all great ideas IMHO. I'm using color/name combos in OP and avatars in Telegram via Octoprint-Telegram plugin personally and being able to use all the same in OF would be awesome :)

Will be supporting the colour field in 1.2 :)

![image](https://user-images.githubusercontent.com/61137988/148196617-d601d319-3d22-4167-a0b3-d42a640c46b6.png)

**Reply by NotExpectedYet** [reply]
![image](https://user-images.githubusercontent.com/61137988/148200839-9839bdd8-d170-4394-9b8c-e602b1bff660.png)
![image](https://user-images.githubusercontent.com/61137988/148200862-88098f0d-c544-4c57-9391-f15f7449f19e.png)

**Reply by NotExpectedYet** [reply]
> Avatars, picking up the color from OP's appearance menu and location tags are all great ideas IMHO. I'm using color/name combos in OP and avatars in Telegram via Octoprint-Telegram plugin personally and being able to use all the same in OF would be awesome :)

What avatars do you speak of?
